server/db_component.cpp

```diff
@@ -228,9 +230,10 @@ int DB::IntegerQuery(string & sql)
 	return rv;
 }
 
-int DB::GetTrackCount()
+int DB::GetTrackCount(string nspace)
```

Comment:
Maybe leave a comment saying that `nspace` has already been properly escaped/cleaned?